azurerm/internal/services/loganalytics/tests/resource_arm_log_analytics_datasource_windows_performance_counter_test.go

```diff
@@ -0,0 +1,235 @@
+		}
+
+		id, err := parse.LogAnalyticsDataSourceID(rs.Primary.ID)
+		if err != nil {
+			return err
+		}
+
+		if resp, err := client.Get(ctx, id.ResourceGroup, id.Workspace, id.Name); err != nil {
+			if !utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("failed to get on LogAnalytics.DataSourcesClient: %+v", err)
+			}
+		}
+
+		return nil
+	}
+
+	return nil
+}
+
+func testAccAzureRMLogAnalyticsDataSourceWindowsPerformanceCounter_basic(data acceptance.TestData) string {
+	template := testAccAzureRMLogAnalyticsDataSourceWindowsPerformanceCounter_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_log_analytics_datasource_windows_performance_counter" "test" {
+  name                = "acctestLADS-WPC-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  workspace_name      = azurerm_log_analytics_workspace.test.name
+  counter_name        = "CPU"
+  object_name         = "CPU"
+  interval_seconds    = 10
+  instance_name       = "*"
+}
+`, template, data.RandomInteger)
+}
+
+func testAccAzureRMLogAnalyticsDataSourceWindowsPerformanceCounter_complete(data acceptance.TestData) string {
+	template := testAccAzureRMLogAnalyticsDataSourceWindowsPerformanceCounter_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_log_analytics_datasource_windows_performance_counter" "test" {
+  name                = "acctestLADS-WPC-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  workspace_name      = azurerm_log_analytics_workspace.test.name
+  counter_name        = "Mem"
+  object_name         = "Mem"
+  interval_seconds    = 20
+  instance_name       = "inst1"
+}
+`, template, data.RandomInteger)
+}
+
+func testAccAzureRMLogAnalyticsDataSourceWindowsPerformanceCounter_requiresImport(data acceptance.TestData) string {
+	template := testAccAzureRMLogAnalyticsDataSourceWindowsPerformanceCounter_basic(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_log_analytics_datasource_windows_performance_counter" "import" {
+  name                = azurerm_log_analytics_datasource_windows_performance_counter.test.name
+  resource_group_name = azurerm_log_analytics_datasource_windows_performance_counter.test.resource_group_name
+  workspace_name      = azurerm_log_analytics_datasource_windows_performance_counter.test.workspace_name
+  counter_name        = azurerm_log_analytics_datasource_windows_performance_counter.test.counter_name
+  object_name         = azurerm_log_analytics_datasource_windows_performance_counter.test.object_name
+  interval_seconds    = azurerm_log_analytics_datasource_windows_performance_counter.test.interval_seconds
+  instance_name       = azurerm_log_analytics_datasource_windows_performance_counter.test.instance_name
+}
+`, template)
+}
+
+func testAccAzureRMLogAnalyticsDataSourceWindowsPerformanceCounter_template(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
```

Comment:
We're adding the package initials to all our test resource groups which helps better track potential test issues through thee resource group.

```suggestion
  name     = "acctestRG-la-%d"
```

Reply:
Will do it.